### ISSUE TITLE
Balloon mapping page missing?

### ISSUE THREAD

**Issue by BeckiC** [publiclab/plots2#807]
### What happened just before the problem occurred
Clicked through from Tools page and got an error.

### Relevant URLs

https://publiclab.org/wiki/balloon-mapping

### PublicLab.org username
Becki

(to help reproduce the issue)

### Browser, version, and operating system

For bug reports, fill out the above template; for feature requests, you can delete the template.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

**Comment by jywarren** [comment]
Yikes, this could be related to
https://github.com/publiclab/plots2/issues/802 which is my top priority
first thing tomorrow. Thanks.

**Comment by jywarren** [comment]
Yes, looks like that's it.

**Comment by jywarren** [comment]
I'm seeing that the record shows: `#<DrupalNode nid: 22, vid: 27209, type: "page", language: "", title: "Balloon & Kite Mapping", uid: 1, status: 1, created: 1292007214, changed: 1471733029, comment: 0, promote: 0, moderate: 0, sticky: 0, tnid: 0, translate: 0, cached_likes: 5, drupal_comments_count: 0, drupal_node_revisions_count: 117, path: "/wiki/balloon-mapping", main_image_id: nil, slug: "--234">`

I believe we'd fixed this record already, so I want to check if there's something resetting it.

**Comment by jywarren** [comment]
Ugh, so there's another record which I moved to "balloon-mapping-2" -- but this seems to be a different error: 

```ruby
ActionView::Template::Error (undefined method `username' for nil:NilClass):
    3: <p>
    4:   <span id="tags">
    5:     <% @node.community_tags.each do |tag| %>
    6:       <span id="tag_<%= tag.id %>" class="label label-primary" data-toggle="tooltip" data-html="true" title="created by <strong><%= tag.author.username %></str
ong> <%= time_ago_in_words(Time.at(tag.date)) %> ago">
    7:         <a href="<%= "/maps" if @node.type == "map" %>/tag/<%= tag.name %>"><%= tag.name %></a>
    8:         <% if current_user && (tag.uid == @node.uid || current_user.role == "admin" || current_user.role == "moderator") %>
    9:           <a class="tag-delete" data-remote="true" href="/tag/delete/<%= @node.id %>/<%= tag.id %>" data-tag-id="<%= tag.id %>">x</a>
  app/views/tag/_tagging.html.erb:6:in `block in _app_views_tag__tagging_html_erb___1417291621532041840_73621280'
  app/views/tag/_tagging.html.erb:5:in `_app_views_tag__tagging_html_erb___1417291621532041840_73621280'
  app/views/wiki/show.html.erb:113:in `_app_views_wiki_show_html_erb___3529977177811974228_68466720'
```

Fix in testing: https://github.com/publiclab/plots2/pull/808

**Comment by jywarren** [comment]
Fixed in #808.
